server/pom.xml

```diff
@@ -31,6 +31,7 @@
     <name>IoTDB Server</name>
     <properties>
         <iotdb.test.skip>false</iotdb.test.skip>
+        <hadoop.version>2.7.3</hadoop.version>
```

Comment:
Why adding this? 
<hadoop.version> is already set in the iotdb-parent pom.xml.

Reply:
If remove this, package dependency will conflict and I don't know why...